src/NovelRT/Ecs/Graphics/DefaultRenderingSystem.cpp

```diff
@@ -255,7 +256,7 @@ namespace NovelRT::Ecs::Graphics
         ResolveGpuResourceCleanup();
         ResolveGpuFutures();
 
-        context->BeginDrawing(NovelRT::Graphics::RGBAColour(0, 0, 255, 255));
+        context->BeginDrawing(_backgroundColour);
```

Comment:
Can we note in this PR that this is a supported feature now? For documentation purposes.

Reply:
Ye I can

Reply:
I don't think this has been done yet? 😅